### PR TITLE
Migrate Bulward to use github.com/kubermatic/utils

### PR DESCRIPTION
**Pull request by jiachengxu**
**What this PR does / why we need it**:
This PR removes utils function and starts to use https://github.com/kubermatic/utils
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/manager/manager.go [hunk 1]
```diff
@@ -28,8 +28,9 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/healthz"
 
+	"github.com/kubermatic/utils/pkg/util"
```
Comment (jiachengxu) [inline pkg/manager/manager.go:31]: @nmiculinic How do we want to sort our import? How about we put kubermatic related import to a dedicated import group?

Comment (nmiculinic) [reply pkg/manager/manager.go:31]: They are like that, aren't they? with `goimports -w github.com/kubermatic` ?

Comment (jiachengxu) [reply pkg/manager/manager.go:31]: yeah, you are right

**Pull request merged**